docs/bucket/notifications/README.md

```diff
@@ -175,11 +176,12 @@ An example configuration for MQTT is shown below:
         "qos": 1,
         "username": "",
         "password": "",
-        "queueDir": ""
+        "queueDir": "",
+        "queueLimit": 0
     }
 }
 ```
-MQTT supports persistent event store in the client-side. The persistent store will backup events when the MQTT broker goes offline and replays it when the broker comes back online. The event store can be configured by setting the directory path in `queueDir` field in the mqtt config. For eg, the `queueDir` can be `/home/events`.
+Minio supports persistent event store. The persistent store will backup events when the MQTT broker goes offline and replays it when the broker comes back online. The event store can be configured by setting the directory path in `queueDir` field in the mqtt config. For eg, the `queueDir` can be `/home/events`.
 
```

Comment:
We should talk about queueLimit as well ?

Reply:
Done